core/src/main/scala/org/bitcoins/core/wallet/utxo/UTXOSpendingInfo.scala

```diff
@@ -57,6 +58,19 @@ sealed abstract class UTXOSpendingInfo {
   def redeemScriptOpt: Option[ScriptPubKey]
 
   def scriptWitnessOpt: Option[ScriptWitness]
+
+  def conditionalPath: ConditionalPath
+}
+
+/** Represents the spending branch being taken in a ScriptPubKey's execution
+  *
```

Comment:
Probably should mention that this is meant to be used in conjunction with `ConditionalScriptPubKey`, and specifically used for control flow in a Script and give an example with with a simple `OP_IF/OP_ELSE` control flow